invenio_rdm_records/fixtures/tasks.py

```diff
@@ -27,6 +36,241 @@ def create_vocabulary_record(service_str, data):
 @shared_task
 def create_demo_record(data):
     """Create a demo record."""
-    service = current_rdm_records.records_service
+    service = current_rdm_records_service
     draft = service.create(data=data, identity=system_identity)
     service.publish(id_=draft.id, identity=system_identity)
+
+
+@shared_task
+def create_demo_community_submission_request(
+        user_id,
+        community_data,
+        drafts_data,
+        n_drafts=1,
+        n_requests=1
+):
+    """Create n_requests community_submission requests for n_drafts drafts."""
+    """Drafts are created in a new community."""
+    # Create a demo community
+    user = current_datastore.get_user(user_id)
```

Comment:
try/catch + nice error if user does not exist? Only if the user was passed as a param: if hardcoded, no need because it should always exist.